python/lsst/ts/ledprojector/config_schema.py

```diff
@@ -0,0 +1,100 @@
+# This file is part of ts_ledprojector.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+# TODO: change github
+# $schema: http://json-schema.org/draft-07/schema#
+# $id: https://github.com/lsst-ts/
+# ts_dimm/blob/master/python/lsst/ts/dimm/config_schema.py
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """
+schema: http://json-schema.org/draft-07/schema#
+title: LEDProjector v1
+description: Schema for LEDController
+type: object
+properties:
+  device_type:
+    description: LabJack model
+    type: string
+    default: T7
+  connection_type:
```

Comment:
I think this could be an enum with TCP, USB as values. https://json-schema.org/understanding-json-schema/reference/enum#enumerated-values